pkg/tms/tms.go

```diff
@@ -443,3 +473,94 @@ func upload(communicationInstance *CommunicationInstance, uploadRequestData pipe
 	defer response.Body.Close()
 	return data, nil
 }
+
+func UnmarshalServiceKey(serviceKeyJson string) (serviceKey serviceKey, err error) {
```

Comment:
[minor]
Strictly saying, the file `tms.go` was dedicated to the client-like methods. The methods added below are more utils-like by their nature and do not fully fit here. But this is minor.

Reply:
maybe we can introduce a utils.go file in the pkg/tms to separate concerns better.

Reply:
done